apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/JobCreationOptions.cs

```diff
@@ -44,5 +46,19 @@ public abstract class JobCreationOptions
         /// </summary>
         /// <seealso cref="BigQueryClient.WithDefaultLocation(string)"/>
         public string JobLocation { get; set; }
+
+        /// <summary>
+        /// The labels associated with the job to be created.
+        /// Labels are key-value pairs.
+        /// You can use these to organize and group your jobs.
+        /// Label keys and values can be no longer than 63 characters,
+        /// can only contain lowercase letters, numeric characters,
+        /// underscores and dashes.International characters are allowed.
+        /// If a label value is <code>null</code>, it won't be added to the job's labels.
+        /// A label value can be <see cref="string.Empty"/>.
+        /// Label keys must start with a letter
+        /// and each label in the list must have a different key.
```

Comment:
Not sure we need to mention about each label having to have a different key - given that it's a dictionary, that would only not be the case if someone provided a very odd comparer or their own dictionary implementation. (I think the fact that it *is* a dictionary is enough.)

Reply:
Agreed, done.